torchci/scripts/check_alerts.py

```diff
@@ -246,15 +252,12 @@ def gen_update_comment(original_body: str, jobs: List[JobStatus]) -> str:
     if len(stopped_failing_jobs) > 0:
         s += "These jobs stopped failing:\n"
         for job in stopped_failing_jobs:
-            s += f"* {job}\n"
+            s += f"* {generate_failed_job_hud_link(job)}\n"
         s += "\n"
     if len(started_failing_jobs) > 0:
         s += "These jobs started failing:\n"
         for job in started_failing_jobs:
-            s += f"* {job}\n"
-    if s:
-        # funky string to make internal alert messaging have code blocks
-        return f"````\n```\n{s}\n```\n````"
```

Comment:
cc @clee2000 since you were the one who originally added code blocks here, is it okay to remove these?